crates/tabby-common/src/config.rs

```diff
@@ -27,10 +27,23 @@ pub struct Config {
 impl Config {
     pub fn load() -> Result<Self> {
         let cfg_path = crate::path::config_file();
-        let mut cfg: Self = serdeconv::from_toml_file(cfg_path.as_path()).context(format!(
-            "Config file '{}' is missing or not valid",
-            cfg_path.display()
-        ))?;
+        if !cfg_path.as_path().exists() {
+            InfoMessage::new(
+                "Tabby config file missing",
+                HeaderFormat::Blue,
+                &[
+                    &format!(
+                        "Warning: Could not find the Tabby configuration at {}",
+                        cfg_path.display()
+                    ),
+                    "Applying default configuration",
+                ],
+            )
+            .print();
```

Comment:
```suggestion
```

No need to generate a info message - or maybe just generate a `debug!` would be sufficient.